### DISCUSSION TITLE
Curso de JavaScript / livro

### DISCUSSION THREAD

**Discussion by Al3xFlores** [frontendbr/forum#1878] [Perguntas e Respostas]
Bom dia deve!!

Estou a procura de indicação para:

Curso de JavaScript na udemy, e 
um livro bom de JavaScript.

Pode ser também conteúdos do YouTube pois tenho conhecimento da base de JavaScript  quero aprofundar mais a linguagem.

**Comment by hannakelly** [comment]
Olá, te indico https://www.udemy.com/share/101WqGBUYbeF5WQHw=/ este da Udemy é maravilhoso.

**Comment by mauvieira** [comment]
Fala Alex, beleza?

Parabéns pela iniciativa, JavaScript é uma linguagem bem legal ;)
Aqui estão alguns recursos que me ajudaram/ajudam muito:

**Livros**:

- [You Don't Know JS Yet](https://github.com/getify/You-Dont-Know-JS)
- [Eloquent JavaScript](https://eloquentjavascript.net/)
- [Clean Code JavaScript](https://github.com/ryanmcdermott/clean-code-javascript)

**Cursos**:

- [JS com TDD na Prática](https://www.udemy.com/course/js-com-tdd-na-pratica/)
- [JavaScript Ninja](https://www.udemy.com/course/curso-javascript-ninja/)
- [Modern JavaScript From The Beginning](https://www.udemy.com/course/modern-javascript-from-the-beginning/)

**YouTube**:

- [The Net Ninja](https://www.youtube.com/c/TheNetNinja/playlists)
- [Programming with Mosh](https://www.youtube.com/c/programmingwithmosh/playlists)

Abraços, bons estudos o/
